scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -134,7 +135,7 @@ object ScroogeSBT extends Plugin {
     // returns Seq[File] - directories that include thrift files
     scroogeUnpackDeps <<= (
       streams,
-      configuration,
+      configuration in Compile,
```

Comment:
You should base the patches off of master. develop branch may have dependencies on unpublished versions of dependencies (util and finagle).